azdev/operations/help/refdoc/common/directives.py

```diff
@@ -37,13 +37,13 @@ def handle_signature(self, sig, signode):
         signode += addnodes.desc_addname(sig, sig)
         return sig
 
-    def needs_arglist(self):  # pylint: disable=no-self-use
+    def needs_arglist(self):
         return False
 
     def add_target_and_index(self, name, sig, signode):
         signode['ids'].append(name)
 
-    def get_index_text(self, modname, name):  # pylint: disable=unused-argument, no-self-use
```

Comment:
Otherwise it raises `azdev/operations/help/refdoc/common/directives.py:46:0: R0022: Useless option value for 'disable', 'no-self-use' was moved to an optional extension, see https://pylint.pycqa.org/en/latest/whatsnew/2/2.14/summary.html#removed-checkers. (useless-option-value)`

Reply:
Pylint apparently doesn't think `no-self-use` is important anymore: https://github.com/pylint-dev/pylint/issues/5502